app/Http/Controllers/Admin/Jexactyl/ApprovalsController.php

```diff
@@ -37,13 +36,15 @@ public function index(): View
 
         return view('admin.jexactyl.approvals', [
             'enabled' => $this->settings->get('jexactyl::approvals:enabled', false),
+            'webhook' => $this->settings->get('jexactyl::approvals:webhook', false),
```

Comment:
Remove the `false` fallback value. If there isn't anything, it'll just show up as empty anyway. (For the webhook_url)